app/views/common/os_selection/_pxe_loader.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= fields_for item do |f| %>
+  <% selected_attr = f.object.new_record? && item.hostgroup.pxe_loader.nil? ? { :selected => @operatingsystem.preferred_loader } : {} -%>
```

Comment:
This needs something like `f.object.new_record? && item.hostgroup.present? && item.hostgroup.pxe_loader.nil?`